src/sentry/tasks/derive_code_mappings.py

```diff
@@ -185,7 +191,7 @@ def set_project_codemappings(
         )
         if not created:
             logger.info(
-                "derive_code_mappings: code mapping already exists",
+                "Code mapping already exists",
```

Comment:
The prefix is already included in the logger name:
https://github.com/getsentry/sentry/blob/e70ab191dcaf43eead5754c69480d7755822493c/src/sentry/tasks/derive_code_mappings.py#L24